repos/keg-components/src/kegComponents.stories.mdx

```diff
@@ -0,0 +1,49 @@
+<!--- Keg-Components.stories.mdx -->
+
+import { Meta } from '@storybook/addon-docs/blocks'
+
+<Meta title="Welcome" />
+
+# Welcome
+
+Keg-Components is a set of cross-platform React and React-Native UI components for building applications.
+The components can be used in both web and native (mobile/desktop) applications, without needing to worry about separate logic or compatibility.
+
+Use these components as the base to build your application upon. This will allow a consistent and reliable way to write components once, then reuse them across multiple platforms
+
+## Install
+Keg-components is [hosted on npm](https://www.npmjs.com/package/@keg-hub/keg-components) and can be installed using any package manager that uses the npm api
+
+With **yarn**
+```bash
+yarn add @keg-hub/keg-components
+```
+With **npm**
+```bash
+npm install @keg-hub/keg-components
+```
+
+## Import / Use 
+
+This library allows importing just what you need or the entire thing. Each component is exported individually to help keep bundle sizes down
+
+**Individually**
+
+```js
+import { Button } from '@keg-hub/keg-components/button'
```

Comment:
Typically this should not be needed. Me mostly do this because of the Keg-Hub setup. 
But to be honest, it's been a while since I tested it without the that.
Might be worth doing, the key point here is, you should not need to do this for a normal app, consuming keg-components